beaker-vue/src/components/panels/FilePanel.vue

```diff
@@ -56,7 +56,16 @@
         <Column field="name" header="Name" class="filename-column" sortable>
           <template #body="slotProps">
             <span :class="[...getFileIconClass(slotProps.data), 'file-icon', slotProps.data.type]"></span>
```

Comment:
Would be cool to have distinct icons for file types we "natively" preview.
Attaching some possibilities.

![file-pdf-color-red-icon](https://github.com/user-attachments/assets/c9119542-8bfe-426a-9a19-ebe80eec20a1)
![pdf-files-icon](https://github.com/user-attachments/assets/64cf911e-c870-46e9-bde0-5dcaf8f02da0)
![xlsx-file-icon](https://github.com/user-attachments/assets/5c74e058-5032-4fa8-8313-345da813a8fc)
![csv-file-icon](https://github.com/user-attachments/assets/6398113c-7c9a-4867-b3a8-ebdfaba7c573)